src/packages/rescue/rat_rescue.py

```diff
@@ -837,3 +784,76 @@ def change(self):
     # TODO: to/from json
     # TODO: track changes
     # TODO: helper method for adding / editing quotes
+
+    def __format__(self, format_spec):
+        """
+        'c' gives the thing colour
+        'a' gives rat assignments
+        '@' gives uuids
+
+        order of format specifiers is NOT sensitive but IS case sensitive!
+        """
+
+        coloured = 'c' in format_spec
+        show_assigned_rats = 'r' in format_spec
+        show_uuid = '@' in format_spec
+
+        buffer = StringIO()
+        buffer.write(f"[{self.board_index}")
+
+        buffer.write(f"@{self.api_id}] " if show_uuid else '] ')
+        buffer.write(F"{self.client}'s case, ")
+        if self.irc_nickname != self.client and self.irc_nickname is not None:
+            buffer.write(f"IRC Nick: {self.irc_nickname!r}, ")
+
+        if self.code_red:
+            base = '(CR '
+            if coloured:
+                buffer.write(bold(color(base, Colors.RED)))
+            else:
+                buffer.write(base)
+
+        if self.platform:
+            base = self.platform.name
+
+            if coloured:
+                if self.platform is Platforms.XB:
+                    buffer.write(color(base, Colors.GREEN))
+                elif self.platform is Platforms.PS:
+                    buffer.write(color(base, Colors.LIGHT_BLUE))
```

Comment:
[Error] Missing case for PC.

Reply:
you know i looked at this a good dozen times, and only now am i seeing that. the Else statement on l824 got be confused...